gradle.properties

```diff
@@ -3,6 +3,7 @@ projectGroup=io.micronaut.problem
 micronautDocsVersion=2.0.0
 micronautVersion=3.2.0
 micronautTestVersion=3.0.4
+serdeVersion=1.0.0-SNAPSHOT
```

Comment:
It would be good to release a Milestone of serde to avoid depending on a milestone.